lib/diagnostic.js

```diff
@@ -47,7 +47,7 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.bind(fire());
 };
```

Comment:
Close! A few issues here:

1) you want this function to `return` something. Right now, it's not returning anything, because `bind` does not do an explicit return
2) the argument that goes inside `bind` needs to be thing that will be the receiver of the action - like a "direct object" - but you have `fire()`, which is a method
3) the method you need to use to make this happen is `.call`, like this:

```js
let fireOnAlderaan = function () {
  return deathStar.fire.call(alderaan);
};
```